samples/subsys/ipc/ipc_service/static_vrings/remote/boards/nrf5340dk_nrf5340_cpunet.overlay

```diff
@@ -42,6 +42,7 @@
 			mbox-names = "rx", "tx";
 			role = "remote";
 			zephyr,priority = <2 PRIO_PREEMPT>;
+			buffer-size = <128>;
```

Comment:
you should not have to define this for remote, buffers are allocated by the host

Reply:
ack